source/Nuke.Common/CI/GitHubActions/Configuration/GitHubActionsCheckoutStep.cs

```diff
@@ -17,7 +17,7 @@ public class GitHubActionsCheckoutStep : GitHubActionsStep
 
         public override void Write(CustomFileWriter writer)
         {
-            writer.WriteLine("- uses: actions/checkout@v2");
```

Comment:
Might be beneficial for the `actions/checkout@v2` part to be public get/set property to allow people to override in their own pipelines when needed if yet another update comes in? 

It would then be just a matter of finding with type and then setting the value to something else.

Reply:
Same goes for the `actions/upload-artifact@v3` too if that makes sense.